src/app/zap-templates/common/ClustersHelper.js

```diff
@@ -166,6 +166,84 @@ function asPutCastType(zclType)
   }
 }
 
+function asChipCallback(atomic)
+{
+  switch (parseInt(atomic.atomicId)) {
+  case 0x00: // nodata / No data
+  case 0x0A: // data24 / 24-bit data
+  case 0x0C: // data40 / 40-bit data
+  case 0x0D: // data48 / 48-bit data
+  case 0x0E: // data56 / 56-bit data
+  case 0x1A: // map24 / 24-bit bitmap
+  case 0x1C: // map40 / 40-bit bitmap
+  case 0x1D: // map48 / 48-bit bitmap
+  case 0x1E: // map56 / 56-bit bitmap
+  case 0x22: // uint24 / Unsigned 24-bit integer
+  case 0x24: // uint40 / Unsigned 40-bit integer
+  case 0x25: // uint48 / Unsigned 48-bit integer
+  case 0x26: // uint56 / Unsigned 56-bit integer
+  case 0x2A: // int24 / Signed 24-bit integer
+  case 0x2C: // int40 / Signed 40-bit integer
+  case 0x2D: // int48 / Signed 48-bit integer
+  case 0x2E: // int56 / Signed 56-bit integer
+  case 0x38: // semi / Semi-precision
+  case 0x39: // single / Single precision
+  case 0x3A: // double / Double precision
+  case 0x49: // struct / Structure
+  case 0x50: // set / Set
+  case 0x51: // bag / Bag
+  case 0xE0: // ToD / Time of day
+  case 0xEA: // bacOID / BACnet OID
+  case 0xF1: // key128 / 128-bit security key
+  case 0xFF: // unk / Unknown
+    return { name : 'Unsupported', type : null };
```

Comment:
should the  name `type` be `cppType` to make usage clear (also "type" feels a bit like a reserved word).

Reply:
`type` is not a reserved keyword in JavaScript. I would prefer to keep `type` since this is what is used across various things exposed to templates and this already confusing enough between `type` and `chipType`.